src/widgets/helper/splitheader.cpp

```diff
@@ -56,7 +57,18 @@ SplitHeader::SplitHeader(Split *_split)
 
         // mode button
         auto mode = layout.emplace<RippleEffectLabel>(this).assign(&this->modeButton);
+        this->addModeItems(mode.getElement());
 
+        QObject::connect(mode.getElement(), &RippleEffectLabel::clicked, this, [this] {
+            QTimer::singleShot(80, [&] {
```

Comment:
http://doc.qt.io/qt-5/qtimer.html#singleShot-6 Check out that function, you'll need to pass `this` as the context parameter